frontend/src/main.js

```diff
@@ -2,4 +2,6 @@ import { createApp } from 'vue'
 import App from './App.vue'
 import router from './router'
 
-createApp(App).use(router).mount('#app')
+const app = createApp(App);
+app.use(router);
+app.mount('#app');
```

Comment:
For context, this was a change I made when testing VuePeerJS, I can revise this back to the one line solution when I resolve the merge conflict.